batch/batch/server/server.py

```diff
@@ -417,7 +417,39 @@ def to_json(self):
 log.info(f'app.root_path = {app.root_path}')
 
 
+authorized_users = [
+    'konradk@broadinstitute.org',
+    'labbott@broadinstitute.org',
+    'dking@broadinstitute.org',
+    'akotlar@broadinstitute.org',
+    'jigold@broadinstitute.org',
+    'ci'
+]
+
+
+def authorized_users_only(fun):
+    def wrapped(*args, **kwargs):
+        if not request.headers.get('User', '') in authorized_users:
```

Comment:
I think our custom headers should be prefixed by `Hail-` and we should use Capitals-With-Dashes which seems standard.  So, `Hail-Internal-Request-None`?

Reply:
done